src/components/Node.tsx

```diff
@@ -55,15 +52,12 @@ export class UnconnectedNode extends React.Component<IExtraProps & INode & INode
             return {
                 nodeID: dbid,
 
-                parent: -1,
+                parent: 1,
                 id: dbid,
-                isentity: false,
-                isleaf: false,
                 isinstance: false,
                 level: 0,
                 mentioncount: 0,
                 name: 'undefined',
-                url: 'un.defi.ned',
```

Comment:
well, hmm, seems like double work to remove it first and add it later, but whatever.